eng/internal/Directory.Build.props

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <Import Project="..\dependencies.props" />
```

Comment:
I'd add an explicit comment on why you aren't/don't want to import the `Directory.Build.props` above this.